src/org/centos/pipeline/PipelineUtils.groovy

```diff
@@ -632,22 +632,19 @@ def executeInContainer(String stageName, String containerName, String script) {
     // env vars from host. We force them in.
     //
     containerEnv = env.getEnvironment().collect { key, value -> return key+'='+value }
-    sh "mkdir -p " + stageName
+    sh "mkdir -p ${stageName}"
     try {
         withEnv(containerEnv) {
             container(containerName) {
-                sh 'pwd'
-                sh 'ls -l /tmp'
                 sh "cp -fv ${WORKSPACE}/fedora.keytab /home/fedora.keytab"
```

Comment:
Could probably do an if stageName = rpmbuild then cp, but not sure that makes it cleaner =)

Reply:
we sure it is only needed by the rpmbuild container?

Reply:
I am pretty sure it is only ever used in the build submission to koji, which is in the rpmbuild stage. The integration and functional tests don't use it. @p3ck none of the compose or booty sanity stages use it, correct?

Reply:
correct, nothing but rpmbuild uses the fedora keytab.